browsers/private/update-tool/versions.mts

```diff
@@ -0,0 +1,39 @@
+interface LatestVersionsPerMilestone {
+  milestone: string;
+  version: string;
+  revision: string;
+}
+
+interface LatestVersionsPerMilestoneResponse {
```

Comment:
Interestingly this is only for Chrome for Testing, but I think it also works for headless etc. Maybe worth adding a comment?


```suggestion
interface ChromeForTestingLatestVersionsPerMilestoneResponse {
```